upup/pkg/fi/cloudup/openstacktasks/floatingip.go

```diff
@@ -131,34 +134,29 @@ func (e *FloatingIP) Find(c *fi.Context) (*FloatingIP, error) {
 	}
 	cloud := c.Cloud.(openstack.OpenstackCloud)
 	if e.LB != nil && e.LB.PortID != nil {
-		// Layer 3
-		fips, err := cloud.ListL3FloatingIPs(l3floatingip.ListOpts{
-			PortID: fi.StringValue(e.LB.PortID),
-		})
+		fip, err := findFipByPortID(cloud, fi.StringValue(e.LB.PortID))
```

Comment:
I do not see how this is going to fix anything to instances which have floatingip attached? This applies only to loadbalancers